izpack-panel/src/main/java/com/izforge/izpack/panels/userpath/UserPathConsolePanel.java

```diff
@@ -43,29 +46,16 @@
  */
 public class UserPathConsolePanel extends AbstractConsolePanel
 {
-    public static final String PATH_VARIABLE;
-    public static final String PATH_PACK_DEPENDS;
-    public static final String PATH_ELEMENT;
-    public static final String USER_PATH_INFO;
-    public static final String USER_PATH_NODIR;
-    public static final String USER_PATH_EXISTS;
+    public static final String PATH_VARIABLE = UserPathPanel.PATH_VARIABLE_NAME;
```

Comment:
Why not using a static import for this constant?

Reply:
Done, using static import now.